internal/api/errors.go

```diff
@@ -226,6 +226,8 @@ func HandleResponseError(err error, w http.ResponseWriter, r *http.Request) {
 			output.Message = e.Message
 			output.Payload.Reasons = e.Reasons
 
+			w.Header().Set("x-sb-error-code", output.ErrorCode)
```

Comment:
```suggestion
```

Might be able to remove this, since output is a `HTTPError` and will have an error code field (weak password set above) and it should also fall through to the case block below and be set there.

Reply:
Hm, but we want to set the header so that API gateway doesn't have to parse the body to find out the error code.